opennms-webapp/src/main/webapp/errors/alarmidnotfound.jsp

```diff
@@ -50,11 +50,15 @@
 
 <h1>Alarm ID Not Found</h1>
 
+<h1>Alarm Cleared or Not Found</h1>
+
 <p>
-  The alarm ID <%=einfe.getBadID()%> is invalid. <%=einfe.getMessage()%>
+  <%=einfe.getMessage()%>. The alarm has been cleared or has an invalid alarm ID.
   <br/>
-  You can re-enter it here or <a href="alarm/list.htm?acktyp=unack">browse all
-  of the alarms</a> to find the alarm you are looking for.
+  Re-enter the alarm ID below or <a href="alarm/list.htm?acktyp=unack">browse all
+   alarms</a> to find the alarm you are looking for.
```

Comment:
I so want to say "find the alarm you seek" as it sounds more dramatic, but that would be silly. :-D